app.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+var productsArray = [];
+var imgPathArray = [];
+var previousImgDisplay = [];
+var img1 = document.getElementById('image1');
+var img2 = document.getElementById('image2');
+var img3 = document.getElementById('image3');
+var body = document.getElementsByTagName('body')[0];
+var totalClicks = 0;
+
+function Products(name, filePath){
+  this.name = name;
+  this.filePath = filePath;
+  this.countShown = 0;
+  this.countClicked = 0;
+  productsArray.push(this);
+  imgPathArray.push(filePath);
+};
+
+var bag = new Products('bag', 'img/bag.jpg');
+var banana = new Products('banana', 'img/banana.jpg');
+var bathroom = new Products('bathroom', 'img/bathroom.jpg');
+var boots = new Products('boots', 'img/boots.jpg');
+var breakfast = new Products('breakfast', 'img/breakfast.jpg');
+var bubblegum = new Products('bubblegum', 'img/bubblegum.jpg');
+var chair = new Products('chair', 'img/chair.jpg');
+var cthulhu = new Products('cthulhu', 'img/cthulhu.jpg');
+var dogDuck = new Products('dog-duck', 'img/dog-duck.jpg');
+var dragon = new Products('dragon', 'img/dragon.jpg');
+var pen = new Products('pen', 'img/pen.jpg');
+var petSweep = new Products('pet-sweep', 'img/pet-sweep.jpg');
+var scissors = new Products('scissors', 'img/scissors.jpg');
+var shark = new Products('shark', 'img/shark.jpg');
+var sweep = new Products('sweep', 'img/sweep.png');
+var tauntaun = new Products('tauntaun', 'img/tauntaun.jpg');
+var unicorn = new Products('unicorn', 'img/unicorn.jpg');
+var usb = new Products('usb', 'img/usb.gif');
+var waterCan = new Products('water-can', 'img/water-can.jpg');
+var wineGlass = new Products('wine-glass', 'img/wine-glass.jpg');
+
+function randomNumGen() {
+  return Math.floor(Math.random() * imgPathArray.length);
+};
+
+function randomImgGen() {
+  var currentImgDisplay = [];
+  while (currentImgDisplay.length < 3){
+    var randomNum = randomNumGen();
+    if (!currentImgDisplay.includes(randomNum) == !previousImgDisplay.includes(randomNum)) {
```

Comment:
== is a comparison operator, this yield a false positive when an image is in both current and previousImgDisplay arrays (!true == !true  -->  false == false --> true).